src/corelibs/U2View/src/ov_msa/MultilineScrollController.cpp

```diff
@@ -308,9 +355,9 @@ void MultilineScrollController::centerViewRow(QPoint maPoint) {
     vScrollBar->setValue(newScreenYOffset);
 }
 
-void MultilineScrollController::centerPoint(const QPoint& maPoint) {
-    centerBase(maPoint.x());
-    centerViewRow(maPoint);
+void MultilineScrollController::_centerPoint(const QPoint& maPoint) {
+    _centerBase(maPoint.x());
```

Comment:
do not use underscore for method names (this style is not used by UGENE).

Reply:
The method are deprecated and will be removed soon
Underscore is used as additional reminder

Reply:
all these methods was removed